train.py

```diff
@@ -71,11 +73,22 @@
 arg_parser.add_argument("--lr_scheduler", type=str, default="none")
 arg_parser.add_argument("--ta_model_path", type=str, default="")
 arg_parser.add_argument("--ta_probability", type=float, default=0.0)
+arg_parser.add_argument("--ta_perturb_amount", type=float, default=0.0)
 
 arg_parser.add_argument("--disable_wandb", dest="enable_wandb", action="store_false")
 arg_parser.set_defaults(enable_wandb=True)
-arg_parser.add_argument("--name", type=str, default=None, help="Manually the run name (e.g., snowy-owl-10); None for automatic naming.")
-arg_parser.add_argument("--unique", dest="unique", action="store_true", help="Make the name unique by appending random digits after the name")
+arg_parser.add_argument(
+    "--name",
+    type=str,
+    default=None,
+    help="Manually the run name (e.g., snowy-owl-10); None for automatic naming.",
+)
+arg_parser.add_argument(
+    "--unique",
+    dest="unique",
+    action="store_true",
+    help="Make the name unique by appending random digits after the name",
+)
```

Comment:
Feel free to remove 80 thru 91 since, like you said, it's not needed now that I know how to use wandb a bit more.  Or I will remove it in the next PR.